utils/nctl/sh/scenarios/chainspecs/slow-node-01.chainspec.toml.override

```diff
@@ -0,0 +1,3 @@
+[core]
+minimum_era_height = 15
+era_duration = "63 seconds"
```

Comment:
Related to failed attempt to improve Highway behavior, pretty sure it can be removed.

Reply:
Removed in 76e484bd6a11048642af57128198153f398d0903